### PR TITLE
updatecheck: Temporarily disable updatecheck test for k8s

### PR DESCRIPTION
**Pull request by keegancsmith**

### CODE REVIEW

File: cmd/frontend/internal/app/pkg/updatecheck/handler_test.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ func TestLatestDockerVersionPushed(t *testing.T) {
 }
 
 func TestLatestKubernetesVersionPushed(t *testing.T) {
+	t.Skip("Temporarily disabled due to issue with 3.13.2. https://sourcegraph.slack.com/archives/CJX299FGE/p1584648681456400")
```
Comment (ggilmore) [inline cmd/frontend/internal/app/pkg/updatecheck/handler_test.go]: Why is this necessary?

**Comment by keegancsmith** [comment]
abandon ship
